component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/backupcode/util/BackupCodeUtil.java

```diff
@@ -443,5 +447,28 @@ public static String getBackupCodeAuthenticatorConfig(String key, String tenantD
                     ERROR_CODE_ERROR_GETTING_CONFIG.getMessage(), e);
         }
     }
+
+    /**
+     * Generate the hash value of the given string.
+     *
+     * @param backupCode String value that needs to hash.
+     * @return Hash value of the backupCode.
+     * @throws BackupCodeException If the algorithms is invalid.
+     */
+    public static String generateHashString(String backupCode) throws BackupCodeException {
```

Comment:
Method name and parameter name are not matching.

Reply:
addressed with [https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/8c6332b5de6e8777c61e430cccb7c65fac331d92](https://github.com/wso2-extensions/identity-outbound-auth-backup-code/pull/6/commits/8c6332b5de6e8777c61e430cccb7c65fac331d92)